src/pattern-library/patterns/tooltips/tooltips.njk

```diff
@@ -1,5 +1,6 @@
-<button class="w-button--icon" data-icon="{{ data.icon }}">
-  <span class="w-tooltip {{ data.modifier }}" role="tooltip">{{
-    data.tooltip
-  }}</span>
-</button>
+<div class="tooltip" data-alignment="{{ data.modifier }}">
+  <button class="fab" aria-labelledby="{{ data.tooltipID }}">
+    {% include "icons/plus.svg" %}  
```

Comment:
`includeRaw` includes without trying to process the SVG with Nunjucks

Reply:
Yeh I tried to work the icon macros and pulling in the custom blocks was a pain in the ass and taking up too much time. We'll sort in integration.